sideloader/postinstall.sh

```diff
@@ -0,0 +1,9 @@
+pip="${VENV}/bin/pip"
+
+cd "${INSTALLDIR}/${REPO}/"
+
+$pip install -e "${INSTALLDIR}/${REPO}/"
+
+springboard="${VENV}/bin/springboard"
+
+$springboard bootstrap
```

Comment:
It clones repos, creates mappings and syncs content. It's safe to run multiple times.